data/json/monsters.json

```diff
@@ -5729,6 +5765,7 @@
     "armor_bash":0,
     "armor_cut":0,
     "vision_day":30,
+    "vision_night":10,
```

Comment:
I think Shia is rare enough for this to be okay.  I've played many, many games and still never run into one.